config/jobs/kubernetes/sig-network/ci-e2e-gce-netd.yaml

```diff
@@ -1096,7 +1096,7 @@ periodics:
       - --ginkgo-parallel=1
       - --provider=gce
       - --test_args=--ginkgo.focus=\[sig-network\].*\[Disruptive\]|\[sig-network\].*\[Serial\]|\[Feature:PerformanceDNS\] --minStartupPods=8
-      - --timeout=120m
+      - --timeout=300m
```

Comment:
@krzyzacy I vaguely remembered kubetest timeout has to be 20 mins less than the bootstrap timeout, is that true anymore?

Reply:
(at least 20min less, so 340m vs 120m is actually valid, depend on what you want to do for cluster down)

Reply:
thanks, I have updated this config file so that all tests have 40 min buff.

Reply:
Thanks, that makes sense.